superset/datasets/schemas.py

```diff
@@ -88,6 +88,18 @@ class DatasetMetricsPutSchema(Schema):
     uuid = fields.UUID(allow_none=True)
 
 
+class FolderSchema(Schema):
+    uuid = fields.UUID()
```

Comment:
@betodealmeida does this UUID need to be generated on the client if it's required here?

Reply:
Right, the client needs to generate a UUID when creating folders. They are only used by tools that want to keep track of external folders and Superset folders.